docs/iota.go/reference/api_get_transactions_to_approve.md

```diff
@@ -0,0 +1,39 @@
+# API -> GetTransactionsToApprove()
+GetTransactionsToApprove does the tip selection via the connected node.  Returns a pair of approved transactions which are chosen randomly after validating the transaction trytes, the signatures and cross-checking for conflicting transactions.  Tip selection is executed by a Random Walk (RW) starting at random point in the given depth, ending up to the pair of selected tips. For more information about tip selection please refer to the whitepaper (http://iotatoken.com/IOTA_Whitepaper.pdf).  The reference option allows to select tips in a way that the reference transaction is being approved too. This is useful for promoting transactions, for example with PromoteTransaction().
```

Comment:
I would maybe link to our docs page, where the current implementation used in IRI is, but then we'll have to update for the new docs 🙈 
https://docs.iota.org/introduction/tangle/tip-selection
So maybe keep the whitepaper and when we're touching the part of the code from which this is generated, we can link to the new site if it's up by then.

Reply:
I've basically copied this from iota.js. I agree that it is way too much detail.